firstvoices/backend/tests/test_serializers.py

```diff
@@ -0,0 +1,128 @@
+import pytest
```

Comment:
(Optional) Should we separate testing of validators in a different file ? I believe their number to increase with more write APIs coming.

Reply:
I wondered this too. I doubt we will have many tests for serializer things other than validators, so maybe at some point we will want to rename this file test_validators.py. I'm going to leave it for now